q2cli/tools.py

```diff
@@ -16,6 +16,26 @@ def tools():
     pass
 
 
+@tools.command(name='export', short_help='Export data.',
+               help="Export data from a QIIME Artifact or Visualization. See "
```

Comment:
Maybe we can just truncate this help text to "Export data from a QIIME Artifact or Visualization.", since any artifact can be exported, and there is no real config or options to provide.